sys/include/sema.h

```diff
@@ -58,92 +78,76 @@ typedef struct {
  * @param[in] value Initial value for the semaphore.
  *
  * @return  0 on success.
- * @return  -EINVAL, if semaphore is invalid.
```

Comment:
Might be problematic, since it is an API change, but since we only have one state after return it might make sense to set the return type to void.